lookout/style/typos/analyzer.py

```diff
@@ -146,17 +171,57 @@ def render_comment_text(self, typo_fix: TypoFix) -> str:
         :param typo_fix: Information about typo fix required to render a comment text.
         :return: string with the generated comment.
         """
-        # TODO: move this logic to template. See FormatAnalyzer.render_comment_text()
-        corrections_line = ", ".join("%s (%d%%)" % (
-            c.token, int(c.confidence * 100)) for c in typo_fix.candidates)
-        return "Possible typo in \"%s\". Suggestions: %s" % (typo_fix.token, corrections_line)
+        file_lines = typo_fix.head_file.content.decode("utf-8", "replace").splitlines()
+        confidences = [candidate.confidence for candidate in typo_fix.identifier_candidates]
+        return self.comment_template.render(
+            identifier=typo_fix.identifier,
+            suggestions=[candidate.token for candidate in typo_fix.identifier_candidates],
+            old_code_line=file_lines[typo_fix.line_number - 1],
+            confidences=confidences,
+            zip=zip,
+            enumerate=enumerate,
```

Comment:
`enumerate` is a built-in. http://jinja.pocoo.org/docs/2.10/templates/#for -> `loop.index`